### DISCUSSION TITLE
Dead project?

### DISCUSSION THREAD

**Discussion by SmoothPlay** [socioboard/Socioboard-5.0#425] [General]
So, it's 2023 now. 

I spent 2 days trying to make this work via Standard Install, behind a reverse proxy (Nginx Proxy Manager) on Ubuntu 22.04.

After getting the server ready with all the dependencies, and no errors occurring during install.

I have all services running with PM2, I have the Frontend and Admin Page Online. And the databases seem to have tables on them so they are connected.

BUT, when trying to sign up for a user, i get the "undefined" error.

I've seen many people reporting this problem but still no one with a solution.

So I have 2 questions:

1- Is anyone still working on the opensource version of socioboard?
2- Does it actually work? Does it work with the API's of the social-media it says it does?

**Comment by TechnologyClassroom** [comment]
1. Doubt it.

2. Doubt it. Doubt it.

[Ferdium](https://github.com/ferdium/ferdium-app) seems to be an active project with a somewhat similar goal. I would try that. I have been following development of this project for some time and you got further than most. I use CLI tools for everything I need at the moment, but I have been on the lookout for projects like this for others on my team. I should probably unsubscribe at this point.

**Reply by SmoothPlay** [reply]
Oh, one more thing, I just installed the Docker version just to make sure and it gives the same error. Hm...

I'll check Ferdium! Thanks for the tip

**Reply by dimmthewitted** [reply]
Did you have problems with version 4 or the API version 5.
I have all APIs working but User and the DBs configured, but then I found this and wondered if its worth my time.

**Comment by TechnologyClassroom** [comment]
You could also try this fork that supposedly got it working as well as it works. It sounds like you already made it that far though: https://github.com/vaughngx4/Socioboard-5.0/tree/development

Twitter/X CLI tool I referenced before: https://github.com/orakaro/rainbowstream

**Reply by SmoothPlay** [reply]
Thanks a lot for your help man, apparently from what I read vaughngx4 also abandoned socioboard, if that guy that worked is ass off did it, it think we  probably should too. This project just can't inspire any confidence considering there is a paid website and almost no updated information or proof that this thing works at a bare minimum.

Thank you for your external suggestions, I'll check those. 

And if someone reads this discussion, I hope it will save you guys the minimum of 2 days of your time to install this thing.

**Comment by moblesmarti** [comment]
This is a lively and interesting project
https://github.com/gitroomhq/postiz-app
